build.gradle

```diff
@@ -1,15 +1,22 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+    /**
+     * The Android Gradle plugin is only required when opening this project stand-alone.
+     * This avoids unnecessary downloads and potential conflicts when the library is included as a
+     * module dependency in an application project.
+     * */
+    if (project == rootProject) {
```

Comment:
Hi @SaeedZhiany - This only applies to the way _React Native_ Android native modules are distributed and consumed. Since this is a regular Android project, you can simply update the Android Gradle plugin version (there is no need to wrap it in a conditional). 👍

Reply:
Hi @friederbluemle 
Yeah I know, but I want to use this library in [react-native-datetimepicker](https://github.com/react-native-community/react-native-datetimepicker) as a better replacement for google date picker module, you can read [this issue](https://github.com/react-native-community/react-native-datetimepicker/issues/16) for more information.

So I think it's maybe necessary, isn't it?

Reply:
If I understand your point correctly, I can use this library by importing it through the dependencies section of gradle like this (`implementation com.wdullaer:materialdatetimepicker:X.Y.Z`), is that right?
then you're right, I had completely forgotten about that in native android development world (this is prons of coding two whole years using react-native😅😅😅)

Reply:
I reverted the changes and just upgrade the AGP version to the latest

Reply:
I reverted the changes and just upgrade the AGP version to the latest,

Thanks for reviewing this

Reply:
Correct, in your project you would simply add a dependency like this: `implementation com.wdullaer:materialdatetimepicker:X.Y.Z` - This pulls a pre-built and packaged .aar file. It won't need or use the Android Gradle plugin defined in this project.